genesyscloud/resource_genesyscloud_webdeployments_configuration_test.go

```diff
@@ -0,0 +1,240 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v56/platformclientv2"
+)
+
+func TestAccResourceWebDeploymentsConfiguration(t *testing.T) {
+	var (
+		configName        = "Test Configuration " + randString(8)
+		configDescription = "Test Configuration description " + randString(32)
+		fullResourceName  = "genesyscloud_webdeployments_configuration.basic"
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
```

Comment:
It's best to have one or more config blocks with different values in order to test that the resource can be successfully updated